src/main/resources/application.yaml

```diff
@@ -47,11 +52,26 @@ bullet:
     message.size.limit.kb: 64
     # The time limit (in seconds) when sending messages to a WebSocket session. It should be positive.
     message.send.time.limit.secs: 10
+  async:
+    # The following configures the asynchronous query result returning of the service. Use this if you want to use the
+    # bullet.endpoint.async endpoint and get results sent to a place of your choice. You must configure responders
+    # below of type com.yahoo.bullet.pubsub.PubSubResponder below.
+    # Enables Asynchronous query result publishing to responders of your choice.
+    enable: false
```

Comment:
We use "enabled" instead of "enable" everywhere else (builtin.rest.enabled, backend.status.enabled, etc)